ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -291,6 +291,10 @@ In AVD 4.0.0, we had "switched" as the default value for `ethernet_interfaces[].
 
 With AVD 5.0.0, the default value for `type` in `ethernet_interfaces` and `port_channel_interfaces` is no longer supported. The `type` key must now be explicitly defined in the input variables if it is needed in the configuration and documentation.
 
+### `router_pim_sparse_mode.ipv4.rp_addresses[].address` is unmarked as a `primary_key`
+
+With AVD 5.0.0, the key `router_pim_sparse_mode.ipv4.rp_addresses[].address` is `not` a `primary_key`.
```

Comment:
A user does not know what this means.

Reply:
Removed these lines.

Reply:
I think we need to update the title and description not removed

Reply:
Updated the title in porting guide. Redefined PR title too.